src/backend/env/config.rs

```diff
@@ -187,6 +192,11 @@ pub const CONFIG: &Config = &Config {
 
     active_user_share_for_minting_promille: 10,
 
+    #[cfg(any(test, feature = "dev"))]
+    difficulty_amplification: 1,
+    #[cfg(not(any(test, feature = "dev")))]
+    difficulty_amplification: 6,
```

Comment:
How did you arrive at 6 for 11 years? Also the poll results by voting power are 15 years.

Reply:
See the extended commit message: https://github.com/TaggrNetwork/Taggr/pull/28/commits/29b667a42fcffbd66af0720ef7a72d15baeb017c

I applied the same logic as on reward proposals: computed the average weighted by VP.

Reply:
I think I need to clarify more: I computed that we should target about 1.3-1.5k tokens per week _at the current_ activity level, then took the last distribution numbers `9125` tokens and divided by the target number, which gives us the required amplification of 6-7. 

Please note that this is all hand-waving and only a temporary solution to curb the minting. We should change this to a proper model as soon as we agree on one.

Reply:
I see, thanks! I was wondering if we should incrementally increase the difficulty to avoid abrupt changes? Wdyt about starting with difficulty 2 and increasing it gradually over several weeks?

Reply:
What would be a good reason for a gradual increase? I consider this as quick fix of a situation spiralling out of control. E.g. if we hit 5% of total supply, a [circuit-breaker](https://github.com/TaggrNetwork/Taggr/blob/main/src/backend/env/mod.rs#L1125) will trigger and Taggr will not mint anything at all. Last week we hit 3.3%.

And most importantly, as with the other suggestion, I would not like to put more complexity into this than absolutely necessary. I think this change is much better than stopping the minting as many would have supported as well. 

Given that we most likely will reduce the weekly numbers even more (and not increase anymore), there is no good reasons to invest time in adding a gradual slow down and keep minting tokens, knowing we're not happy about how they get distributed. But please let me know if you feel strongly about it.

Reply:
> And most importantly, as with the other suggestion, I would not like to put more complexity into this than absolutely necessary.

I didn't mean to do it programmatically. Instead, we could increase the factor with every release. The motivation would be to soften the impact and also observe if the posting behavior will change.

Reply:
And if the oosting behavior changes we revert? 😀

Reply:
> And if the posting behavior changes we revert?

There will be more time for discussion. I always prefer smaller steps and more iterations with abrupt changes it is difficult to predict what can go wrong.

Reply:
Actually I thought more about this and am very confident now that it will not have any prolonged negative effects. 

First of all, I’m convinced that the current distribution is wrong and would like to enforce more merit based distributions. For example, as Ulan pointed out somewhere, I will create quarterly reward proposals for all contributions of different people and would encourage others do the same. 

Content only should not create whales in governance, IMO because we all agree in the meantime that quality content is a very relative concept and difficult to measure. 

Then, economically, by reducing the supply, at the equally staying demand, the price will just multiply by the difficult increase over time. 

And finally, those who feel (until the price increase) that it’s not worth their time to use Taggr, they IMO should leave anyway. 

With that I feel very confident doing this change because I’m convinced that it fixes things. Then the question if a quick fix is better than a slow one is kind of obvious to me.

Reply:
Oh and: we already crossed the previous Fridays number of tokens to be minted and were have 2 days to mint in front of us!


Also it looks like the proposed time based distribution I proposed will be approved without modifications. But that one is very easy to implement so we can have it very soon, sooner than phasing out the current difficulty by artificial halvings.